package.json

```diff
@@ -53,7 +53,14 @@
       "fast-loops@<1.1.4": ">=1.1.4",
       "fast-xml-parser@<4.4.1": ">=4.4.1",
       "elliptic@<6.5.7": ">=6.5.7",
```

Comment:
I know it's not part of this PR, but since I started researching if we are actually using these packages as deps or not, I discovered that we can update `elliptic` in the shared package instead of overriding it here.

Reply:
Same for:
- electron-updater

Reply:
Done

Reply:
But I still see elliptic being listed here. It should be removed from this file and any other place where we override it